lib/dashing/cli.rb

```diff
@@ -59,17 +59,11 @@ def install(gist_id)
     def start(*args)
       port_option = args.include?('-p') ? '' : ' -p 3030'
       args = args.join(' ')
-      command = "bundle exec thin -R config.ru start#{port_option} #{args}"
+      command = "bundle exec puma #{port_option} #{args} config.ru"
       command.prepend "export JOB_PATH=#{options[:job_path]}; " if options[:job_path]
       run_command(command)
     end
 
-    desc "stop", "Stops the thin server"
-    def stop
-      command = "bundle exec thin stop"
-      run_command(command)
-    end
```

Comment:
You can't just remove the ability to stop the thing...

Please add in support for stopping Puma as per this Gist: https://gist.github.com/joelmoss/2470666